docs/cep/query-guide/functions/memcache/memcache.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Memcache
+---
+
+The `memcache` function provides a persistent cache per tenant. It is similar to [cache](../cache/), but this function has a time-to-live (ttl) parameter.
+
+## Features
+
+The following functions are allowed:
+
+- `memcache:get("key")`       - gets a value per given key
+- `memcache:put("key", "value", ttl)` - puts "key" and "value" with ttl, where ttl set by default 5000L (5 seconds)
+
+## Syntax
+
+Memcache uses the following syntax:
+
+```sql
+memcache:get(<String> key) 
+memcache:put(<String> key, <String> value, <Long> ttl)
+memcache:get(<String> key) <String> memcache:put(<String> key, <String> value)
```

Comment:
It seems typo. Must be `<String> memcache:put(<String> key, <String> value)`

Reply:
I wondered. I just copy/pasted this from the Jira ticket.